app/models/ability.rb

```diff
@@ -0,0 +1,3 @@
+class Ability
+  include CanCan::Ability
+end
```

Comment:
- [ ] I noticed that your `Ability` class does not contain any set of rules for the user - post interaction. Please define rules that state the following 👇 
![image](https://user-images.githubusercontent.com/46839548/222841763-a45c1e5d-e302-4e4f-9f84-396a1a1bd2bc.png)